contrib/gitian-build.py

```diff
@@ -184,6 +177,16 @@ def main():
         if not 'LXC_GUEST_IP' in os.environ.keys():
             os.environ['LXC_GUEST_IP'] = '10.0.3.5'
 
+    if args.setup:
+        setup()
+        exit(0)
```

Comment:
Should be `sys.exit(0)` :-)

Surprised that `flake8` didn't complain about that!

Reply:
Thanks!